From 9d346ca900eb982b533f72a585ea10357b4e3006 Mon Sep 17 00:00:00 2001
From: Hiroki Negishi <hiroki.negishi.bx@renesas.com>
Date: Mon, 13 Nov 2017 11:51:17 +0900
Subject: [PATCH 12/23] Revert "Revert "arm64: dts: r8a7795: Add MFIS Lock
 device node""

This reverts commit 1d4a318e4a40 ("Revert "arm64: dts: r8a7795:
Add MFIS Lock device node"")

Signed-off-by: Hiroki Negishi <hiroki.negishi.bx@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a7795.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795.dtsi b/arch/arm64/boot/dts/renesas/r8a7795.dtsi
index ffbfab5..c56a65e 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7795.dtsi
@@ -880,6 +880,13 @@
 			status = "okay";
 		};
 
+		mfis_lock: mfis-lock@e62600c0 {
+			compatible =    "renesas,mfis-lock-r8a7795",
+					"renesas,mfis-lock";
+			reg = <0 0xe62600c0 0 0x0020>;
+			status = "okay";
+		};
+
 		pfc: pfc@e6060000 {
 			compatible = "renesas,pfc-r8a7795";
 			reg = <0 0xe6060000 0 0x50c>;
-- 
1.9.1

